WordPress/WordPressTest/FeatureFlagTest.swift

```diff
@@ -25,8 +25,6 @@ class FeatureFlagTest: XCTestCase {
     func testEnsureDisabledFeaturesInProduction() {
         Build.withCurrent(.AppStore) {
             expect(FeatureFlag.People.enabled).to(beFalse())
-            expect(FeatureFlag.Plans.enabled).to(beFalse())
```

Comment:
Ah, looks like you're the author of those originally, up to you, but I'd be OK with removing them along with this PR.